src/Hazelcast.Net.Linq.Async/Hazelcast.Net.Linq.Async.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0;net6.0</TargetFrameworks>
+        <RootNamespace>Hazelcast.Linq</RootNamespace>
+    </PropertyGroup>
```

Comment:
Since this is a brand new project, can we enable nullable checks?
I.e. add a `<Nullable>enable</Nullable>` in that property group?
It will trigger warnings, but I think it's better to handle them right at the start.

Reply:
Enabled and handled. Please, see my comment above.